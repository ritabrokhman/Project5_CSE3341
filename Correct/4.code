procedure t4 is
	object a;
	object b;
	object c;
	procedure A(object x) is
		b : x;
		c : x;
		print(x['default']);
	end
begin
	a = new object('default', 1);
	b = new object('default', 1);
	c = new object('default', 1);
	a['default'] = 99;
	begin A(a);
end
