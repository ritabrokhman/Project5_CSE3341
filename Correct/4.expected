gc:1
gc:2
gc:3
gc:2
gc:1
99
gc:0
